serverless/pages/custom-roles.mdx

```diff
@@ -0,0 +1,108 @@
+---
+slug: /serverless/custom-roles
+title: Custom roles
+description: Create and manage roles that grant privileges within your project.
+tags: [ 'serverless', 'Elasticsearch', 'Observability', 'Security' ]
+---
+
+{/* <DocIf condition={"((serverlessCustomRoles))" === "false"}>
+<DocBadgeComingSoon>Coming soon</DocBadgeComingSoon>
+</DocIf> */}
+
+{/* TO-DO: Change feature flag before merging */}
+{/* <DocIf condition={"((serverlessCustomRoles))" === "true"}> */}
+<DocBadge template="technical preview" />
+This content applies to: <DocBadge template="elasticsearch" slug="/serverless/elasticsearch/what-is-elasticsearch-serverless" /> <DocBadge template="observability" slug="/serverless/observability/what-is-observability-serverless" /> <DocBadge template="security" slug="/serverless/security/what-is-security-serverless" />
+
+The built-in <DocLink slug="/serverless/general/assign-user-roles" section="organization-level-roles">organization-level roles</DocLink> and <DocLink slug="/serverless/general/assign-user-roles" section="instance-access-roles">instance access roles</DocLink> are great for getting started with ((serverless-full)), and for system administrators who do not need more restrictive access.
+{/* With so many features, it's not possible to ship built-in roles that accommodate everyone's needs.
+This is where custom roles come in. */}
+{/* Derived from https://www.elastic.co/guide/en/kibana/current/tutorial-secure-access-to-kibana.html#_roles */}
+
+{/*
+TBD: Accurate for serverless? Which types of users have access to this UI?
+The `manage_security` cluster privilege is required to manage custom roles.
+*/}
+As an administrator, however, you have the ability to create your own roles to describe exactly the kind of access your users should have.
+For example, you might create a marketing_user role, which you then assign to all users in your marketing department.
+This role would grant access to all of the necessary data and features for this team to be successful, without granting them access they don't require.
+{/* Derived from https://www.elastic.co/guide/en/kibana/current/tutorial-secure-access-to-kibana.html */}
+
+By default, custom roles grant the same access as the `Viewer` instance access role with regards to ((ecloud)) privileges.
+If you want to change this behavior, you must create your custom role by using the ((es)) create role API.
+{/* TBD: Is it the Elasticsearch or Kibana role management API in this case? Add link when API docs are available. */}
```

Comment:
@jeramysoucy does the Control Plane Role API allow users to change this access level? I don't expect this would be done via the Kibana or ES APIs.

Reply:
> does the Control Plane Role API allow users to change this access level? I don't expect this would be done via the Kibana or ES APIs.

@legrego @lcawl This behavior is fixed - custom role assignments will always contain the `Viewer` cloud role and there is currently no way to override or bypass this. With multiple role assignments, additional roles can be assigned, so you could elevate a user with a custom role from `Viewer` to `Editor` or `Admin`, but this would inherently also bestow the stack privileges of that elevated role.

Does that make sense?

Reply:
Thanks for the information!  I've updated this section via b16dd3cbe8e58b3966dcb75f52240d1bdf236d77